packages/radix-ui-themes/src/components/theme-panel.tsx

```diff
@@ -135,7 +135,7 @@ const ThemePanelImpl = React.forwardRef<ThemePanelImplElement, ThemePanelImplPro
       function handleKeydown(event: KeyboardEvent) {
         const isModifierActive = event.altKey || event.ctrlKey || event.shiftKey || event.metaKey;
         const isKeyboardInputActive = document.activeElement?.closest(keyboardInputElement);
-        const isKeyT = event.key.toUpperCase() === 'T' && !isModifierActive;
+        const isKeyT = event.key?.toUpperCase() === 'T' && !isModifierActive;
```

Comment:
TS doesn't think `key` could be undefined, but I'm definitely seeing keydown events without `key` property when using keyboard early on page load

Reply:
wow, weird, I didn't think this was possible… We never check this in primitives, don't feel like that should be needed…

Reply:
Was also surprised. I get this on the first keypress in sink
<img width="1026" alt="image" src="https://github.com/radix-ui/themes/assets/8441036/2c52ce79-ddb3-4b30-b085-3e3eab6cf3a9">

Reply:
weird, I don't see this.